package.json

```diff
@@ -16,24 +16,31 @@
   },
   "files": [
     "docs/",
-    "rules/",
-    "processors/",
-    "index.js"
+    "src/",
+    "lib/"
   ],
+  "main": "lib/",
   "engines": {
     "node": ">=6"
   },
   "peerDependencies": {
     "eslint": ">=5"
   },
   "scripts": {
+    "postinstall": "yarn build",
```

Comment:
Is auto-build after install helpful in this case?
Unlike in the Jest repo, you'll be able to run `yarn jest` just fine.
In development, `yarn build --watch` will initially build everything anyway (afaik that's what Babel CLI `watch` does)

Reply:
Lint will fail since we run with our own rules. And jest runs lint, so it'd fail

Reply:
Okay, didn't consider that 👍

Reply:
@SimenB `postInstall` runs when this package is being `npm/yarn` installed.

We can't assume that the user has `yarn` or any of the `devDependencies`.

I think we should revert it for now.

Reply:
Maybe the intention was `prepublishOnly`?